Sources/Remote Logging/RemoteLoggingExports.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+
```

Comment:
```suggestion
```

I _think_ we shouldn't need to explicitly import Foundation in here? Or am I missing something? 🤔